src/sentry/static/sentry/app/bootstrap.jsx

```diff
@@ -35,17 +35,16 @@ Sentry.init({
     new Tracing({
       tracingOrigins: ['localhost', 'sentry.io', /^\//],
     }),
+    new TransactionActivity(),
   ],
 });
 
-Sentry.configureScope(scope => {
```

Comment:
Is this no longer the preferred way to set a tag?

Reply:
It's 🤷‍♂ 
`Sentry.setTag` is short for just setting a tag.
We exposed those functions a while ago for convenience.